app/src/main/java/com/eveningoutpost/dexdrip/ui/dialog/ColorPreferenceDialog.java

```diff
@@ -29,7 +29,6 @@ public static void pick(final Activity activity, final String pref, final String
         }
         picker.setColor(color);
         picker.showAlpha(true);
-        picker.showHex(false);
```

Comment:
use `picker.showHex(Home.get_engineering_mode());` here to enable it only in engineering mode.